ingestion-beam/src/main/java/com/mozilla/telemetry/options/SinkOptions.java

```diff
@@ -91,9 +91,9 @@
 
   void setIncludeStackTrace(Boolean value);
 
-  @Description("Fixed window duration. Defaults to 10m. Allowed formats are: "
```

Comment:
these two aren't value providers, why are they losing their default value from the doc string?

Reply:
Because default gets automatically injected into the help output. This one looks like:

```
  --windowDuration=<String>
    Default: 10m
    Fixed window duration. Allowed formats are: Ns (for seconds, example: 5s),
    Nm (for minutes, example: 12m), Nh (for hours, example: 2h).
```

That may have been a change in a more recent version of beam.

Reply:
awesome!